org.lflang/src/org/lflang/generator/ts/TSImportPreambleGenerator.kt

```diff
@@ -67,36 +67,29 @@ class TSImportPreambleGenerator(
     }
 
     private fun generateDefaultImports(): String {
-        return with(PrependOperator) {
-            """
-                |// Code generated by the Lingua Franca compiler from:
-                |// file:/${filePath.toString()}
-                $DEFAULT_IMPORTS
-                |
-            """.trimMargin()
-        }
+        return """
```

Comment:
I'm not sure I understand why it is possible to do away with the `with` clause here?

Reply:
That's because the prepend operator is not used here (that's the `..` operator on String). DEFAULT_IMPORTS is written to take advantage of `trimMargin`. Looking at this, it might be cleaner to change that, because this code relies on the fact that DEFAULT_IMPORTS has the same indentation as the other text blocks that include it...